rotkehlchen/tasks/calendar.py

```diff
@@ -262,13 +297,51 @@ def maybe_create_locked_crv_reminders(self) -> None:
             error_msg='Failed to add the CRV lock period end reminders',
         )
 
+    def maybe_create_airdrop_claim_reminder(self) -> None:
+        """Create reminders for airdrop claim deadlines."""
+        data = check_airdrops(
+            addresses=self.blockchain_accounts.get(SupportedBlockchain.ETHEREUM),
+            database=self.database,
+            data_dir=self.database.user_data_dir,
+        )
+
+        calendar_entries: list[int] = []
+        for address, airdrops in data.items():
+            for airdrop_name, airdrop_info in airdrops.items():
+                if (
+                    airdrop_info['claimed'] is True or
+                    'cutoff_time' not in airdrop_info or
+                    (cutoff_time := Timestamp(airdrop_info['cutoff_time'])) <= ts_now()
+                ):  # Skip if already claimed, unknown cutoff, or past cutoff
+                    continue
```

Comment:
question: if we haven't claimed and you create the reminder but then you claim the calendar entry won't be deleted. Would make sense to delete the calendar entry and reminders here?